.github/workflows/ui-test.yml

```diff
@@ -87,6 +87,7 @@ jobs:
 
       - name: Update
         run: |
+          docker exec -t prestashop sh -c "echo 'memory_limit=1024M' > /usr/local/etc/php/conf.d/zz-memory-limit.ini"
```

Comment:
We take this change in this PR, but once #1397 is merged, we should revert to 512Mo or whatever the default is.
This will allow us to discover other similar issues in the future